### PR TITLE
Module 1: Linter Set up for HTML & CSS

### PR DESCRIPTION
**Pull request by ProgramKingAbel**
Set up Linter Working Environment  and HomePage

Configured Linters - Stylelint, LightHouse and Webhint.
Developed homepage Markup and Styling.
Edit professional README.md file
Installed NPM and JSON dependencies
Added MIT LICENSE.TXT file
Fixed broken MIT link in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,185 @@
...
+I would like to thank the Microverse Community, My stand up team and coding partners who we work tirelessly to make it possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
```
Comment (iambenkis) [inline README.md]: - [x] Your README file looks very professional 🤙 , however I noticed that the link attached to the MIT license is broken , increase the beauty of your README file by fixing this link with a MIT license.
- [x] You could find it[ here](https://choosealicense.com/licenses/mit/)

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @ProgramKingAbel   
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] Good job opening your first Pull Request however it could be better if your PR have a descriptive title telling you what changes are inside. It will make it more professional 👍 .
- [x] Also your Request should have a short summary describing introduced changes in more detail. Kindly add them on the places showed at the image below

Capture


- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ProgramKingAbel) [reply README.md:184]: Working Environment set up with Linters - Stylelint, LightHouse and Webhint.
Developed homepage Markup and Styling.
Edit professional README.md file 
Installed NPM and JSON dependencies

**Comment by ProgramKingAbel** [comment]
Added LICENSE.TXT file 
Fixed broken License link in README.md

File: README.md (hunk 1 above)
Comment (ProgramKingAbel) [reply README.md]: Hello @iambenkis Following the comment on your review, I have added a descriptive summary of my PR as well as added an MIT LICENSE.TXT file to the repository and fixed the broken MIT license broken link in my README.MD. Thank you for the guidance.

**Comment by ProgramKingAbel** [comment]
***@***.*** <https://github.com/iambenkis> I appreciate the Review
feedback. Following that, I added the descriptive title and summary to my
PR as well as added an MIT LICENSE.TXT file to my work and fixed the broken
MIT license link that was in my README.md file. I hope my corrections
satisfy the professional standard that you have advised. Thank you.*

**Review by osamaashraf6 — APPROVED** [review]
Hi @ProgramKingAbel 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Congratulations!GIF

## Highlights
- No linter errors present
- The PR has a proper title and a descriptive body 👍

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**